update-extension-dependencies.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Purpose: Updates pom.xml files with symbolic dependencies to extensions to enforce a consistent build order.
+
+set -e -u -o pipefail
+shopt -s failglob
+
+echo ''
+echo 'Building bom-descriptor-json...'
+echo ''
+#mvn -D quickly -T0.8C -f independent-projects/arc
+#mvn -D quickly -T0.8C -f bom/application
+#mvn -D quickly -T0.8C -f core
+#mvn -D quickly -T0.8C -f extensions
+mvn clean package -f devtools/bom-descriptor-json
+
+DEP_TEMPLATE='        <dependency>
```

Comment:
General idea is to avoid additional (temporary) files, so everything is stored in variables.